langkit/lkt_lowering.py

```diff
@@ -1407,6 +1410,10 @@ def lower_base_field(
                 not annotations.lazy,
                 'Parse fields cannot be lazy'
             )
+            check_source_language(
```

Comment:
Same comment as before

Reply:
This particular area is full of similar existing checks: I would prefer to transfer them to Liblktlang in one pass rather that just do it now for this one. Is it ok for you?

Reply:
As discussed live: I’ll take care of porting existing checks in a later PR (this is my next job wrt Lkt)